casepro/contacts/models.py

```diff
@@ -190,7 +190,7 @@ def get_or_create_from_urn(cls, org, urn, name=None):
         """
         contact = cls.objects.filter(urns__contains=[urn]).first()
         if not contact:
-            contact = cls.objects.create(org=org, name=name, urns=[urn], is_stub=True)
+            contact = cls.objects.create(org=org, name=name, urns=[urn], is_stub=False)
```

Comment:
This should be ok. A stub contact happens when a new message references a contact who hasn't yet been synced from the backend. We mark the contact as a stub to tell the message handler not to handle their messages yet, as message labelling rules may reference fields/groups of the contact which aren't yet known. In this case we're not running any rules.